cyder/management/commands/dhcp_migrate.py

```diff
@@ -98,8 +91,7 @@ def create_subnet(subnet_id, name, subnet, netmask, status, vlan_id):
                       "WHERE vlan_id = %s" % vlan_id):
         vlan_id, vlan_name, vlan_number = cursor.fetchone()
         vlan = Vlan.objects.get(name=vlan_name)
-    vrf = Vrf.objects.get(name=('VoIP' if network_str in voip_networks
-                                else 'Legacy'))
+    vrf = Vrf.objects.get(name='Legacy')
```

Comment:
Just to be clear... there are no plans to run `dhcp_migrate` after merging this branch, right? So these changes are purely superficial. Is there any real reason to maintain `dhcp_migrate` and `dns_migrate` at this point? Even if we keep them updated, I doubt anyone will be testing them with any regularity.